### DISCUSSION TITLE
p-value > .999

### DISCUSSION THREAD

**Discussion by Filippo-94** [easystats/correlation#247] [Q&A] [answered]
if I run

correlation(mtcars)

there are there a lot of > .999 p-value, why?
For example:
qsec and gear > .999

but if I run
correlation(mtcars, select=c("qsec", "gear"))

the result is 0.243 instead of > .999

<img width="832" alt="mtcars" src="https://user-images.githubusercontent.com/61939360/182484501-74b30f99-5982-43aa-ac3d-6a310336a8cb.png">

**Comment by bwiernik** [comment, marked as answer]
The reason for this is that a Holm p-value adjustment is applied by default. Set `p_adjust = "none"` to not adjust p values. Or, in general, I recommend ignoring the p values and focusing on the CIs instead.

**Reply by Filippo-94** [reply]
Really a simple thing, p_adjust=“none” solved.
I’ll focus on CIs, thank you for the advice! Filippo
